docs/rfcs/0005-video-resources.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+parent: RFCs
+nav_order: 5
+has_toc: true
+---
+
+# 0005: OCW Video Resources
+{: .no_toc }
+
+## Table of Contents
+{: .no_toc .text-delta }
+
+- Table of Contents
+{:toc}
+
+### Abstract
+Parse the `course_embedded_media` section of OCW master.json files.  Search for Youtube videos within that section.
+If found, add the video as a resource, including the thumbnail image and transcript text.
+
+
+### Acceptance Criteria
+
+- [ ] Each Youtube video found for an OCW will be saved as a `ContentFile` resource in the database
+- [ ] The resource will include a thumbnail image and transcript text if found.
+- [ ] The Elasticsearch index will include the resource along with the thumbnail and text.
+
+
+### Architecture Changes
+
+The `ContentFile` model will need an additional nullable field, `image_src`, to store the value of the video thumbnail if it exists.  The existing `content` attribute can be used to hold the transcript text.
+The current ETL process for OCW resources will need to be modified to process the `course_embedded_media` section:
+  - Youtube videos in that section will have a `title` & `id` of "Video-YouTube-Stream", with a `media_location` equal to the Youtube video id.
+  - Once a Youtube video is identified, check for the existence of a `Video` object with the same `video_id`
+  - If a matching video is found, copy that video's `image_src` and `transcript` values to the resource's `image_src` and `content` fields.
+  - If a matching `Video` object is NOT found:
```

Comment:
So this sounds like we don't intend for these files to be searchable individually on Open, is that correct?

Reply:
Good point. @mbertrand how about we index all and filter out the rest for ocw ?

Reply:
That is correct, only the video would show up in search results, not the video's thumbnail and transcript file (though the content of the transcript file would be used to find relevant results since it's saved as part of the video resource).  @ferdi I'm not sure if there's a point to indexing the supporting files unless you think we want users to get these returned in results at some point in the future.

Reply:
@mbertrand sure, got it.

Reply:
@rhysyngsun should I merge this yet or would you like to take another look?

Reply:
@mbertrand lgtm!